lib/online_order.rb

```diff
@@ -0,0 +1,90 @@
+require "csv"
+require "ap"
+require_relative "order.rb"
+require_relative "customer.rb"
+
+module Grocery
+  class OnlineOrder < Order
+    attr_accessor :customer, :online_order_id, :products, :fulfillment_status
+
+      @customer = Customer.find(id)
+      @products = products
+      @online_order_id = online_order_id
+      @fulfillment_status = fulfillment_status
```

Comment:
Instead of saving the order id and the product list yourself, you should call `super` and have the `initialize` method for `Order` do that work for you.